### PR TITLE
feat: moving coroutines hands-on to docs

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

**Comment by qwwdfsad** [comment]
@danil-pavlov I have no idea what this review is about and what I should review.

Could you please drop me a line about what you are trying to achieve here and what parts I should look at?

**Comment by danil-pavlov** [comment]
@qwwdfsad Sorry, we're just moving this [hands-on lab](https://play.kotlinlang.org/hands-on/Introduction%20to%20Coroutines%20and%20Channels/01_Introduction) to the coroutines documentation, and our internal instruction says that we need to get your approval before merging anything to this repo;)
There were no errors when building staging for this change, I checked

**Review by qwwdfsad — APPROVED** [review]

**Pull request merged**
